test/unit/org/apache/cassandra/tools/ToolRunner.java

```diff
@@ -52,15 +50,17 @@
     protected static final Logger logger = LoggerFactory.getLogger(ToolRunner.class);
 
     private static final ImmutableList<String> DEFAULT_CLEANERS = ImmutableList.of("(?im)^picked up.*\\R");
-    
+    private static final String[] EMPTY_STRING_ARRAY = new String[0];
```

Comment:
Looks like this is only used once. Do you need to make it a class constant?

Reply:
used once in a method, but done each call; this caused a intellj warning so switched to this to make the class clear of warnings

Reply:
sgtm